sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -1104,6 +1107,15 @@ public CaseDataDto saveCase(CaseDataDto dto, boolean handleChanges) throws Valid
 
 		caze = fillOrBuildEntity(dto, caze);
 
+		final ContactReferenceDto fromContactReference = dto.getFromContactReference();
```

Comment:
The DTOs ContactDto and CaseDataDto don't have a list of samples, and I have not seen any example where we have/map the many to one associations to the DTO level.
That's why I thought it's best if I keep that the same and handle the association of the samples to the new case within the BE.
It is possible to move it to the CaseFaceadeEJb#buildFromContact - but that means i need to add the samples to the ContactDto and map that every time even though in all other cases it is not used.

Reply:
I think I'd prefer a separate setSampleAssociations method in the case facade that can be called once during the case creation process which basically does what you're currently doing in the saveCase method, because right now, the new fromContactReference field seems to be practically useless after the case has been created.

Reply:
ok, extracted it to separate method